config/locales/ru/admin.yml

```diff
@@ -266,32 +266,31 @@ ru:
     metadata:
       key: "Название"
       value: "Значение"
-      default_metadata: "Метаданные По Умолчанию"
-      generated_jsonld: "Сгенерированные Структурированные Данные"
+      default_metadata: "Метаданные по умолчанию"
+      generated_jsonld: "Сгенерированные структурированные данные"
```

Comment:
These are titles of sections, it would make sense for them to be title case as in the English translation

Reply:
it's unconventional to capitalise articles in Russian (like by, at etc.)
I changed structured data to capital letters.